chart_tests/test_templates/git_sync_scheduler/git-sync-existing-claim.yaml

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+dags:
+ persistence:
+  enabled: true
+  existingClaim: test-claim
```

Comment:
@ashb I think that keeping these as yaml files will make sense as this scales (imagine if we need to test 20 seperate configurations for one scenario, we'd have to maintain 20 dictionaries within a single python file that would be really verbose). It also could be used in documentation later.

@kaxil WDYT?

Reply:
That's two hypotheticals - we can do that when we need it.

Right now it will be easier to understand living right along side the test assertions

Reply:
If we want to have the documentation automatically tested, we can save some examples to a YAML file. However, this does not prevent you from trying to keep the full configurations in one place in most cases.

I guess it won't scale because we want to test the Helm Chart in a lot of different configurations. Very soon we will have more configuration files than test files. Each file will probably only be used in one test so that will be difficult to maintain.

On the other hand, if we keep our configurations in a Python file, we can store them locally within the test_ * function. If you open the test, you can see all input parameters and assertions without changing context.  This allows you to review all tests from A to Z much faster.

Reply:
+1

Reply:
@ashb @mik-laj @kaxil  I moved the configurations into the python files, but I think there could be value here in keeping them as yaml strings as it gives a better 1:1 comparison to an actual launch of the helm chart. Would like feedback though.

Reply:
It looks good to me now.